src/rubrix/server/tasks/text_classification/api/model.py

```diff
@@ -489,7 +489,12 @@ class TextClassificationQuery(BaseModel):
     status: List[TaskStatus] = Field(default_factory=list)
     predicted: Optional[PredictionStatus] = Field(default=None, nullable=True)
 
-    def as_elasticsearch(self) -> Dict[str, Any]:
+    only_uncovered: bool = Field(
+        default=False,
+        description="If enabled, filter records that are not affected by defined rules",
+    )
+
+    def as_elasticsearch(self, rules: List[LabelingRule]) -> Dict[str, Any]:
```

Comment:
Maybe I would provide a default here, just in case `service.py` is not the only place where `as_elasticsearch` is called, and to not break calls of this method.
```suggestion
    def as_elasticsearch(self, rules: Optional[List[LabelingRule]] = None) -> Dict[str, Any]:
```

Reply:
better using `uncovered_by_rules`  as part of query param